src/main/java/org/opensearch/sdk/handlers/ExtensionsRestRequestHandler.java

```diff
@@ -32,13 +36,15 @@
 public class ExtensionsRestRequestHandler {
     private static final Logger logger = LogManager.getLogger(ExtensionsRestRequestHandler.class);
     private final ExtensionRestPathRegistry extensionRestPathRegistry;
+    private NamedXContentRegistry namedXContentRegistry;
```

Comment:
This needs to be the "wrapper" `SDKNamedXContentRegistry` that doesn't change (and make it `final`).  We will use `getRegistry()` on each Rest call to fetch the most recent registry.